microsoft/atlas-design
# Input & Textarea components

Task: task-531896

Link: preview-343

Adding input and textarea components.

## Testing

- [Input](https://design.docs.microsoft.com/pulls/343/components/input.html)
- [Textarea](https://design.docs.microsoft.com/pulls/343/components/textarea.html)
- [Label](https://design.docs.microsoft.com/pulls/343/components/label.html)
- [Help](https://design.docs.microsoft.com/pulls/343/components/help.html)

1. Visit the pages linked above.
2. Click through the input/textarea components, ensure different states are looking good.
